src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
 
+enum Shape {
+  triangle = 'triangle', circle = 'circle', rectangle = 'rectangle'
+}
```

Comment:
```suggestion
enum Shape {
  Triangle = 'triangle',
  Circle = 'circle',
  Rectangle = 'rectangle'
}

```
correct enum formatting :arrow_up: